tests/cluster/eh/test_eh_saving.py

```diff
@@ -2,21 +2,21 @@
 import unittest
 from datetime import datetime, timedelta, timezone
 
-from atc_tools.time import dt_utc
 from pyspark.sql import DataFrame
 from pyspark.sql import functions as f
-
-from atc import Configurator
-from atc.delta import DeltaHandle
-from atc.eh import EventHubCapture, EventHubJsonPublisher
-from atc.eh.EventHubCaptureExtractor import EventHubCaptureExtractor
-from atc.etl import Transformer
-from atc.functions import init_dbutils
-from atc.orchestrators import EhJsonToDeltaOrchestrator
-from atc.spark import Spark
+from spetlrtools.time import dt_utc
+
+from spetlr import Configurator
+from spetlr.delta import DeltaHandle
+from spetlr.eh import EventHubCapture, EventHubJsonPublisher
+from spetlr.eh.EventHubCaptureExtractor import EventHubCaptureExtractor
+from spetlr.etl import Transformer
+from spetlr.functions import init_dbutils
+from spetlr.orchestrators import EhJsonToDeltaOrchestrator
+from spetlr.spark import Spark
 from tests.cluster.values import resourceName
 
-from .AtcEh import AtcEh
+from .SpetlrEh import SpetlrEh
```

Comment:
consider using absolute path like the rest of the imports?

Reply:
let's do this later